### ISSUE TITLE
Code Table Request - Add 

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#6263]
## Instructions

_This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions._

_Reviewing documentation before proceeding will result in a more enjoyable experience._

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

_**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

_**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

_**Attribute data type** If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)

_**Attribute controlled values** If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate.

_**Attribute units** If numberical values should be accompanied by units, provide a link to the appropriate units table.

_**Context**: Describe why this new value is necessary and existing values are not._

_**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

_**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

_**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions._

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by campmlc** [comment]
See #6228

**Comment by dustymc** [comment]
Is "environmental: e[nvironmental]" absolutely necessary? 

I think "environmental" is a method - DNA is DNA and that's what's being tested for, no?

**Comment by Jegelewicz** [comment]
Agree with @dustymc and suggest changing the value to `environmental: DNA`.

**Comment by dustymc** [comment]
I don't think that's an agreement!

I suggested that "environmental" is part of method.

The "thing" is just DNA.

The test will detect it whether it's in the environment or not.

Method should not be embedded in the terms.

**Comment by Jegelewicz** [comment]
> Method should not be embedded in the terms.

then [environmental: PFAS](https://github.com/ArctosDB/arctos/issues/6265) should cause issue too?

I guess I would argue that the "thing" is eDNA because I fear that using DNA on it's own will confuse users of the data.

**Comment by campmlc** [comment]
Agree with @Jegelewicz . There are too many sources of DNA. We need this term to explicitly reference eDNA, to make it discoverable without confounding the attribute with other sources of DNA, such as a liver. Plus, it needs to be under the "environmental" category. You aren't testing an organism. You are testing a body of water, or soil. 
This term is widely used among researchers and the collections community. The point is to make the term discoverable to people who are looking for it, not to hide it under our own semantic labels.

**Comment by campmlc** [comment]
As we have said multiple times before, using these attributes this way is providing a place to put these data RIGHT NOW - because we have these data, they are on hold, this is delaying cataloging and delaying research by people who need to use them. These discussions have been going on since last year. If we can get SOMETHING online, we can get feedback from the user community and we are hoping this will lead to real funding for something better. Yes, there is always something better. I spent three years of my life developing something I thought would be a truly innovative and useable solution for this exact problem only to find out that was also supposedly merely temporary. So let's just create this solution, please, and re-evaluate once we have some data and some dedicated funding.

**Comment by Jegelewicz** [comment]
> You aren't testing an organism. You are testing a body of water, or soil.

but I can see a time when you could also be testing an organism and find some other organism's DNA? Will we then need organism: DNA or some way to say "not environmental: DNA"? This also applies to PFAS and any other "environmental" things because you could certainly test a mouse for PFAS

Our other categories create a kind of hierarchy, but environmental doesn't. It just means we will end up with many ways to find "detected DNA" so that nobody can find all of the detected DNA. I suggest just DNA. But we really do need @KyndallH and/or @megulbranson to weigh in.

**Comment by campmlc** [comment]
eDNA is a thing, and part of an emerging paradigm in genomics research,
only just starting to impact natural history collections. It needs it's own
category, separate from everything else, otherwise researchers will be
unable to discover we have these samples.

**Comment by campmlc** [comment]
It needs its own category also because it is a radically different type of
evidence for an occurrence, which will affect identifications and
attributes. We will need to be able to find these records.
https://www.nhm.ac.uk/discover/what-is-environmental-dna-edna.html

**Comment by Jegelewicz** [comment]
Found in GBIF slack

Just found out about a UN science summit that includes a session to discuss biodiversity and eDNA standards, with an emphasis on freshwater, September 12 - 29: https://sciencesummitunga78.sched.com/event/1NY6U
The description emphasizes an interest in developing new ISO standards.  It appears to be free and virtual, and many other sessions look interesting too.

sciencesummitunga78.sched.comsciencesummitunga78.sched.com
[Science Summit at United Nations General Assembly (UNGA78): [VIRTUAL] Towards inclusivity in interna...](https://sciencesummitunga78.sched.com/event/1NY6U)
View more about this event at Science Summit at United Nations General Assembly (UNGA78)

**Comment by campmlc** [comment]
This really needs to be "eDNA". I can be "environmental: eDNA". 
https://static.sched.com/hosted_files/sciencesummitunga78/18/Hunter_UN%20Science%20Summit_eDNA%20best%20practices_standards_10Sept23.pptx?_gl=1*i7de5n*_ga*NzE4MDM4MDE2LjE2OTI3NDU2MTU.*_ga_XH5XM35VHB*MTY5NTMxNzU5Ny4yLjEuMTY5NTMxNzczMC4xMi4wLjA.

**Comment by happiah-madson** [comment]
The way that we handle this in our current database is to have a sample type = environmental sample and then the child extract is DNA. But I also agree with @campmlc that eDNA is a known entity in the research world and is highly sought so making it easier to find is useful.

> but I can see a time when you could also be testing an organism and find some other organism's DNA? 

@Jegelewicz this isn't the same thing. We already do this all the time (esp in our lab; Dan studies microbial endosymbionts). The environmental in eDNA is abiotic (soil, water, air) not any environment.

**Comment by happiah-madson** [comment]
Also, this may be helpful? (Via Abby Benson, who I work with at OBIS/USGS)

https://docs.gbif.org/publishing-dna-derived-data/en/#environmental-dna-as-a-source-for-dna-derived-occurrence-data

The Standardizing Marine Biological Data group has been grappling with eDNA and we could reach out to them as a resource, if that's useful?

**Comment by happiah-madson** [comment]
> The environmental in eDNA is abiotic (soil, water, air) not any environment.

Okay, the link I just sent immediately refuted my point 😆: eDNA is thus a sample type, not a method, including DNA derived from any environmental sample rather than from the capture and sequencing of a targeted individual. Such sample types includes water, soil, sediment and air, but also gut content samples and tissue (plant/animal) where the host DNA is not targeted ([Taberlet et al. 2018](https://doi.org/10.1093/oso/9780198767220.001.0001)).

But also reiterates that eDNA is a sample type, not a method!

**Comment by campmlc** [comment]
Yes, I agree, that's why it needs to be "eDNA" as a value in the code table!

Mariel Campbell   (she/her)
Collection Manager, Division of Genomic Resources
Museum of Southwestern  Biology

Shipping Address:
Museum of Southwestern Biology
Division of Genomic Resources
1 University of New Mexico
MSC03 2020
Albuquerque, NM 87131

(505) 277-7808 office
email: ***@***.***


________________________________

**Comment by dustymc** [comment]
>eDNA is thus a sample type, 

Exactly! This discussion can **only** pertain to "things which an be detected" - this code table provides value for eg https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#detected. If there's some way to detect (or fail to) environmental DNA while excluding - uhh, extra-environmental?!? - DNA then this could be useful, if not then this still looks like a square peg being hammered into a round hole.

**Comment by happiah-madson** [comment]
But you can collected an eDNA sample (ie, filter a ton of water from a place, or just collect a bunch of water from a place and freeze it) with the _intention_ of it being used for eDNA work (ie, like how we can collect liver tissue for genomic work) and that's still functioning as a sample type?

**Comment by Jegelewicz** [comment]
I would think that water would be the identification. You could use [geosample](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#geosample) as the part, but the definition doesn't really fit. Maybe we would need some new part name for a jug of water - liquid sample?

**Comment by dustymc** [comment]
>intention of it being used for eDNA work

YES!

cataloged item == water
part == filter paper

this attribute== "we've  [tested/detected/whatever] eDNA - not some specific primer or anything, definitely not nonenvironmental DNA, we've just [tested/detected/whatever] eDNA"

?????????

I can see how it's possible to test for DNA (not sure I'd understand _**why**_ anyone would test for random DNA, but I guess I don't have to....) but - well, it gets really weird about there and I don't get it....

**Comment by happiah-madson** [comment]
![Screen Shot 2023-09-21 at 4 22 43 PM](https://github.com/ArctosDB/arctos/assets/47900639/1c32ca7d-0bed-4f41-9e3e-52845bb8070e)

sometimes you are looking for a specific species in an eDNA sample (ie, for invasive species monitoring) and sometimes you aren't and are looking for as much stuff as you can find (ie, for figuring whats in a place). So yes, people DO test for "random" DNA.

**Comment by mkoo** [comment]
Not to gum up the discussion but what collection type is using this? (I'm seeking some clarity with regard to Dusty's comment above). I am on board with the nice definition and schematic that Hannah provides-- eDNA is generally a environmental sample where DNA is detected so water, soil, air, etc is the actual part. So I can see a collection type of eDNA samples but I am struggling to see how this relates to a mammal specimen.

If you are using eDNA to create a species observation (we do this in the disease portal) this seems like it goes somewhere like part or method of identification or somewhere else...

or is the mammal fur or some part of it being subjected to the eDNA process to find its closest friends, etc?

**Comment by happiah-madson** [comment]
OGL has a small number materials with a of sample type of "environmental sample." They all have a 'scientific name' of not determined and are things like seawater filtrate, biofilm, water or sediment samples, a piece of a deep sea chimney...etc.

**Comment by dustymc** [comment]
>"environmental sample." 

Record https://arctos.database.museum/guid/UAM:Env:65 is an environmental sample, the part ("thing to which one could stick a barcode") is https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#filter_paper.

**Comment by campmlc** [comment]
Yes, this has been done at UAM as a separate environmental sample collection, and at MSB we have requested and MSB:Env for the same reason.. But this is an analogous situation to a host/parasite system, and that it causing the confusion. 
This issue is a request #6263 is a request to add "eDNA" to the examined detected code table which would be used for the Env collection type, for this particular term. It would not be used in a mammal collection. But essentially this is the same usage as 
"examined for" and "detected" records attribute in a host collection, but it is soil, water etc that is being examined.

We also need the part name, "eDNA", which could be added to the same record, in the same way we have part name "ectoparasite", which is added to a mammal host record.

Then ideally, in the best of all possible worlds, any taxonomic units which are identified in the DNA screening should be given their own records and linked via relationships.  But that could be hundreds. Alternatively, we add multiple identifications to the Env record.

BTW I can't find any of the Alaska lake water samples that tested positive for frog DNA because logged in a public user, I can't access the Attributes view.

For @happiah-madson, since they have so few, can't they still use A string ID with eDNA as part, without setting up a separate collection?

**Comment by Jegelewicz** [comment]
There is no need for a separate collection. OGL can do exactly what UAM has done in their current collection.

We have geosample so a request to add the part environmental sample seems fair to use when the sample isn't something more specific like biofilm, filter paper, or sediment (geosample). But we can request that later if needed. As it is right now, I don't think that OGL would have use for the examined/detected attribute requested here.

**Comment by campmlc** [comment]
I see, "OGL:Genomic" works like "Env". 
So if the identification is "water" or "soil", then "examined/detected attributes would be appropriate.
